packages/web-worker/test/notifier.test.ts

```diff
@@ -82,6 +82,43 @@ describe('worker notifier', () => {
     Bugsnag.start(testConfig)
     expect(Bugsnag.isStarted()).toBe(true)
   })
+
+  describe('session management', () => {
+    it('successfully starts a session', (done) => {
+      const Bugsnag = getBugsnag()
+      Bugsnag.start(API_KEY)
+      Bugsnag.startSession()
```

Comment:
It might be worth checking that `startSession` is what actually caused the request, e.g.:

```suggestion
      Bugsnag.start(API_KEY)

      expect(typedGlobal.fetch).not.toHaveBeenCalled()

      Bugsnag.startSession()
```